### ISSUE TITLE
Bulk export fails due to missing user

### ISSUE THREAD

**Issue by kaakaa** [mattermost/mattermost-server#11659]
#### Summary
Bulk export fails with the following error log.

```
Error: SqlUserStore.Get: Unable to find the user., user_id=XXXXXXXXXXXXXXXXX
```

The cause of this issue is little complex.
If executing the deleting user command, [PermanentDeleteUser](https://github.com/mattermost/mattermost-server/blob/master/app/user.go#L1387) doesn't delete any reactions that deleting user add. So reactions that the deleted user add are left.
After that, when executing bulk export, it collects all reactions and try to get user who add a reaction. It results in error of bulk export.

#### Steps to reproduce
1. Create dummy_user
2. dummy_user add a reaction to a post
3. Delete dummy_user permanently by `mattermost` command
4. Execute bulk export

#### Expected behavior
Bulk export results in success.

#### Observed behavior (that appears unintentional) 
Failed.

#### Possible fixes

1. Ignore errors that occur [when the user who adds reaction cannot be found](https://github.com/mattermost/mattermost-server/blob/956c8930f55d26b867cd262d0b56453edba64a29/app/export.go#L404), and dispose that reaction from output
2. When deleting a user, delete togetcher all reactions that are added by the user

**Comment by amyblais** [comment]
Hi @kaakaa, do you know if this is a new issue and what version did you see this on?

**Comment by kaakaa** [comment]
@amyblais 
We encountered this issue on v5.11.0, and I confirm steps to reproduce  on `master` branch.
This is the first time for us to execute `export` command, so we don't know whether or not this is new issue.

**Comment by amyblais** [comment]
I tried to ask our team about this but didn't get feedback. However, I opened a ticket for this because this has clear reproduction steps: https://mattermost.atlassian.net/browse/MM-17224.

**Comment by amyblais** [comment]
Closing as this will be fixed in upcoming v5.14 release.
